src/components/App/App.tsx

```diff
@@ -524,6 +528,9 @@ export default class App extends React.Component<AppProps, AppState> {
         }
       );
     });
+    socket.on('REC:roomCapacity', (data: number) => {
```

Comment:
yeah rather than networking the room capacity value let's just make these react app accessible config values? (prefix with REACT_APP)

Reply:
If we didn't broadcast the room capacity, wouldn't we have to broadcast if it's a sub room or not instead?

Reply:
Actually after looking at this again this is only needed for the display on the member list right? If we don't want/need to display the max then we don't need to send it to the client at all.